Tasks/UseRubyVersion/task.json

```diff
@@ -0,0 +1,58 @@
+{
+    "id": "33C63B11-352B-45A2-BA1B-54CB568A29CA",
+    "name": "UseRubyVersion",
+    "friendlyName": "Use Ruby Version",
+    "description": "Retrieves the specified version of Ruby from the tool cache. Optionally add it to PATH.",
+    "helpMarkDown": "",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 133,
+        "Patch": 0
+    },
+    "preview": true,
+    "demands": [],
+    "instanceNameFormat": "Use Ruby $(versionSpec)",
+    "inputs": [
+        {
+            "name": "versionSpec",
+            "type": "string",
+            "label": "Version spec",
+            "required": true,
+            "defaultValue": ">= 2.4",
+            "helpMarkDown": "Version range or exact version of a Ruby version to use."
+        },
+        {
+            "name": "outputVariable",
+            "type": "string",
+            "label": "Output variable",
```

Comment:
A task can produce more than one output variable, provide a nicer friendly name for this variable - e.g "Ruby distribution path"